### DISCUSSION TITLE
Elvox 6209?

### DISCUSSION THREAD

**Discussion by briped** [PricelessToolkit/ESPBell-LITE#1] [Q&A]
My intercom is an Elvox 6209, which I've been wanting to do exactly what you've done, but lacking the knowledge.

I found [this manual](https://www.manualslib.com/manual/1204392/Elvox-Petrarca-6209.html) for it.

Here's several pictures of the intercom.

![image](https://github.com/PricelessToolkit/ESPBell-LITE/assets/4194602/68b6366b-b703-473d-b3b1-cfcf15b6c867)
![image](https://github.com/PricelessToolkit/ESPBell-LITE/assets/4194602/2ac38e83-e486-412c-9866-c402b2452863)
![image](https://github.com/PricelessToolkit/ESPBell-LITE/assets/4194602/5ef948db-102a-4c1c-8dfc-da7d6c06fece)
![image](https://github.com/PricelessToolkit/ESPBell-LITE/assets/4194602/ab5a5522-ade1-4b81-9303-7b2f6c3ff05f)

**Comment by PricelessToolkit** [comment]
![290991142-2ac38e83-e486-412c-9866-c402b2452863](https://github.com/PricelessToolkit/ESPBell-LITE/assets/81021972/421479ae-8850-4ace-ad38-98edb445f666)
I am not sure and it is hard to say something without a schematic or without testing with a multimeter. 1. if just hacking together you can try and connect the input optocoupler to "A- and CA" if the red wires go to the "doorbell loudspeaker". 2. you can connect SSR wires to "Lock button directly". And i hope you understand that I cannot give any guarantee that it will work.
